pytest_jira.py

```diff
@@ -181,10 +205,12 @@ def pytest_configure(config):
         "When 'run' is False, the test will be skipped prior to execution. "
         "See https://github.com/rhevm-qe-automation/pytest_jira"
     )
-
     if config.getvalue("jira") and config.getvalue('jira_url'):
         jira_plugin = JiraHooks(config.getvalue('jira_url'),
                                 config.getvalue('jira_username'),
-                                config.getvalue('jira_password'))
-        ok = config.pluginmanager.register(jira_plugin, "jira_plugin")
-        assert ok
+                                config.getvalue('jira_password'),
+                                config.getvalue('jira_verify'))
+        if jira_plugin.is_connected():
+            # if connection to jira fails, plugin won't be loaded
+            ok = config.pluginmanager.register(jira_plugin)
+            assert ok
```

Comment:
The inner condition (is_connected) has logger output on line n. 45.
And do we want warning for the outer condition? (missing "--jira" parameter)